tools/test-tools/package.json

```diff
@@ -34,14 +35,14 @@
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.51.0",
 		"@fluidframework/eslint-config-fluid": "^5.6.0",
-		"@types/mocha": "^10.0.10",
+		"@types/mocha": "^9.1.1",
```

Comment:
We use mocha 9 types with mocha 10 runtime? That seems like a mistake.

Reply:
Yeah, addressing separately in https://github.com/microsoft/FluidFramework/pull/23404. I looked at the changes in `@types/mocha` since it moved to v10 and it doesn't seem like we were missing anything critical or using anything badly mistyped.